packages/blockchain-link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.1.31-beta.1
+
+-   chore(utils): move Throttler util from blockchain-link to utils (78673bd14c)
```

Comment:
I had hard time here trying to figure out a good fitting commit message when the change was across two published packages.